tests/matlab_test_data_collectors/python_testers/makeLine_test.py

```diff
@@ -18,17 +18,20 @@ def test_makeLine():
         logging.log(logging.INFO, i)
 
         filepath = os.path.join(collected_values_folder, f'{i:06d}.mat')
-        recorded_data = loadmat(filepath)
+        recorded_data = loadmat(filepath, simplify_cells=True)
 
-        params = recorded_data['params'][0]
+        params = recorded_data['params']
         if len(params) == 4:
             Nx, Ny, startpoint, endpoint = params
```

Comment:
I guess makeline is not Cartesian?

Reply:
It seems make line is working with integer cooridinates in pixel space and not float coordinates in Cartesian space is that correct?

Reply:
We can add this point to the issue I opened about refactoring makeline. Someone had already expressed interest in refactoring it.

Reply:
Alright, let's leave this change to the issue you opened.